x/rollup/keeper/msg_server.go

```diff
@@ -239,7 +244,7 @@ func ValidateBasic(m *rollupv1.InitiateWithdrawalRequest) error {
 	}
 	// Check if the gas limit is within the allowed range.
 	gasLimit := new(big.Int).SetBytes(m.GasLimit).Uint64() // size=24 (0x18), offset=64 (0x40)
-	if gasLimit < params.TxGas || gasLimit > params.MaxGasLimit {
+	if gasLimit < MinGasLimit || gasLimit > MaxGasLimit {
 		return fmt.Errorf("gas limit must be between %d and %d: %d", params.TxGas, params.MaxGasLimit, gasLimit)
```

Comment:
Can you use the constants instead of the `params.` values in the error message creation as well?